rubicon_ml/sklearn/pipeline.py

```diff
@@ -82,10 +82,15 @@ def fit(self, X, y=None, tags=None, log_fit_params=True, **fit_params):
         fit_params : dict, optional
             Additional keyword arguments to be passed to
             `sklearn.pipeline.Pipeline.fit()`.
+        experiment: None
```

Comment:
`experiment: rubicon_ml.experiment.client.Experiment, optional` 

also in `score`